Automattic/jetpack
# Contact Form Block: Use a less strict comparison for Atomic compat

Fixes #22696

## Changes proposed in this Pull Request:

Uses a less strict comparison when inspecting for `template-canvas.php` for better compatibility on Atomic.

## Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

See reproduction details in https://github.com/Automattic/jetpack/issues/22696#issuecomment-1195074753

1. Create a new WoA dev site.
2. Add a contact form to a FSE template.
3. Choose "Redirect to another webpage" as the "On submission" option, and enter the URL.
4. Manually apply the change in the pull request to `wp-content/plugins/jetpack/modules/contact-form/grunion-contact-form.php`
5. Navigate to the FSE template on the frontend and submit the form.
6. Verify you're redirected to the URL when the form is submitted.
